### DISCUSSION TITLE
why does the mimic_admission only has 256878 patients,but mimic_core has 382278 patients.

### DISCUSSION THREAD

**Discussion by pike8801** [MIT-LCP/mimic-code#1281] [MIMIC-IV]
why does the mimic_admission only has 256878 distinct subject_id,but mimic_core has 382278 distinct subject_id?

**Comment by marymlucas** [comment]
Just taking a stab in the dark here, but the missing IDs might be patients who were discharged from the ER and not admitted into the hospital.
